pro/src/new_components/OffererHasBeenSoftDelete/OffererHasBeenSoftDelete.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+import styles from './OffererHasBeenSoftDelete.module.scss'
+
+const OffererHasBeenSoftDelete = (): JSX.Element => {
```

Comment:
```suggestion
const OffererHasBeenSoftDeleted = (): JSX.Element => {
```

On pourrait pas avoir un nom du type `SoftDeletedOffererWarning` (ou `Information`, `Message`, `View`...) ? Je trouve que ça irait mieux à un nom de composant qui s'occupe d'afficher un message